strategies.py

```diff
@@ -86,7 +93,8 @@ def __init__(self, network, seconds_per_move=5, simulations_per_move=0,
         self.root = None
         self.result = 0
         self.result_string = None
-        self.resign_threshold = -abs(resign_threshold)
+        self.resign_threshold = resign_threshold or FLAGS.resign_threshold
+        self.timed_match = timed_match
```

Comment:
consider timed_match or self.num_readouts == 0

Reply:
Added an assert to prevent cases where neither time / readout settings are configured.